pkg/runctl/run_manager.go

```diff
@@ -278,6 +284,43 @@ func (c *runManager) copyPipelineCloneSecretToRunNamespace(ctx *runContext, secr
 	return names[0], nil
 }
 
+func (c *runManager) copyElasticSearchCredentialToRunNamespace(ctx *runContext, secretHelper secrets.SecretHelper) (string, error) {
```

Comment:
Add docu, especially return parameters

Reply:
Should I add it before function definition of `copyElasticSearchCredentialToRunNamespace`?